DCCpp_Uno/src/DCCpp.h

```diff
@@ -69,6 +86,8 @@ Part of DCC++ BASE STATION for the Arduino
   #define DIRECTION_MOTOR_CHANNEL_PIN_A 12
   #define DIRECTION_MOTOR_CHANNEL_PIN_B 13
 
+  #define CURRENT_CONVERSION_FACTOR 2.96  // ((5/1024)/1.65) * 1000 
```

Comment:
Is that the standard for defines in C++? Doesn't that hurt readability? Happy to do it though. So instead of this:

#define fred1 10  // blah
#define fred2 20  // more blah
#define fred3 30  // less blah

we want this?

// blah
#define fred1 10
// more blah
#define fred2 20  
// less blah
#define fred3 30

Reply:
It's more common to comment the line above a define since it can also be used for doxygen or similar automatic documentation generation tools.